### ISSUE TITLE
Incorrect syntax error for materialized view

### ISSUE THREAD

**Issue by AceFire6** [sbdchd/squawk#646]
I get a syntax error that seems incorrect when checking the SQL to add a materialized view. The SQL runs correct on a PSQL 16 server.

View SQL

```sql
CREATE MATERIALIZED VIEW "public"."user_countries_view" 
AS (
    select distinct "client_id", "address"->>'country' as "country" 
    from "users" where "users"."address"->>'country' IS NOT NULL
);
```

Running `squawk ./add_mv.sql
```
error[syntax-error]: expected SELECT, TABLE, or VALUES statement, got PAREN_SELECT
 --> add_mv.sql:1:186
  |
1 | ...y" from "users" where "users"."address"->>'country' IS NOT NULL);
  |                                                                    ^
  |

Find detailed examples and solutions for each rule at https://squawkhq.com/docs/rules
Found 1 issue in 1 file (checked 1 source file)
```

I'm running v2.25.1 but I've tested this on 2.23.0 and 2.24.0 with the same results.
```
> squawk --version
squawk 2.25.1
```

**Comment by sbdchd** [comment]
Thank you for the report!

It should be fixed in the latest version!

https://github.com/sbdchd/squawk/releases/tag/v2.26.0

**Comment by AceFire6** [comment]
Thank you for the quick resolution 🙌
